other/concat-exports.js

```diff
@@ -0,0 +1,88 @@
+const appendToFile = require('fs').appendFile
```

Comment:
Are you getting a warning with this? I had to switch to `appendFileSync` to get rid of it.